data/sample.json

```diff
@@ -0,0 +1,18 @@
+{
+    "start_height": 204100,
```

Comment:
Why we don't use data from bitcoind docker?

Reply:
Cause the LC is eventually going to be part of Native chain and bitcoind node can have different states when relayer has not completely synced them, or relayer is down for some duration.

Reply:
I mean for bitcoind snapshot data.

Reply:
It can be of use when doing integration testing and list of blocks is too huge if startHeight is large number.

But I feel bitcoind snapshot should be kept separate from the light client, cause even if needed we can export the bitcoind data into a json with a simple script.

Maybe implement either in another PR?

Reply:
So I mean we can give the optional.

Reply:
This is good if have a light client for bitcoind snapshot data 👍🏼 .

Reply:
We can merge it, but please document how we are going to use it:
* if this is a sample data - I think it can only be used for unit tests.
* for functional or system tests, we need bitcoind snapshot, and then run bitcoind.
So, let's document the flow.

Reply:
https://github.com/orgs/gonative-cc/projects/1/views/1?pane=issue&itemId=92038385